src/components/InputBook.js

```diff
@@ -1,6 +1,17 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { booksActions } from '../redux/books/books';
 
 function InputBook() {
+  const dispatch = useDispatch();
+  const handleAddBook = () => {
+    dispatch(booksActions.addBook({
+      id: 1,
+      title: 'Awa and Other plays',
+      author: 'Ndorrh Oswald',
+
+    }));
```

Comment:
**This is for the next reviewer**
- [ ] Hi it's like there is a misunderstanding. I just realized that for this milestone the student was just supposed to do the reducers and actions so, This part is for the next milestone and can be ignored for this PR. Sorry for the misunderstanding.